base_layer/core/src/core_key_manager/handle.rs

```diff
@@ -380,35 +386,42 @@ where TBackend: KeyManagerBackend<PublicKey> + 'static
     async fn get_metadata_signature_ephemeral_commitment(
         &self,
         nonce_id: &KeyId<PublicKey>,
+        range_proof_type: RangeProofType,
     ) -> Result<Commitment, TransactionError> {
         (*self.core_key_manager_inner)
             .read()
             .await
-            .get_metadata_signature_ephemeral_commitment(nonce_id)
+            .get_metadata_signature_ephemeral_commitment(nonce_id, range_proof_type)
             .await
     }
 
     async fn get_metadata_signature(
         &self,
-        value_as_private_key: &PrivateKey,
         spending_key_id: &KeyId<PublicKey>,
-        sender_offset_private_key: &PrivateKey,
-        nonce_a: &PrivateKey,
-        nonce_b: &PrivateKey,
-        nonce_x: &PrivateKey,
-        challenge_bytes: &[u8; 32],
+        value_as_private_key: &PrivateKey,
+        ephemeral_commitment_nonce_id: &KeyId<PublicKey>,
```

Comment:
I'm generally not in favour of having type aliases, but since we already have a PublicKey type alias, I would suggest making a type alias for `KeyId<PublicKey>` to eliminate this generic, Perhaps `type StoredKey = KeyId<PublicKey>`

Reply:
Or StoredKeyId, or TariKeyId